javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,140 @@
+// function getFullname(firstname, surname) {
+//   return firstname + " " + surname;
+// }
+
+// const fullname1 = getFullname("Benjamin", "Hughes");
+
+// const fullname2 = getFullname("Nina", "Lin");
+
+// console.log(fullname1);
+// console.log(fullname2);
+
+//formal fullname
+function getFullname(firstname, surname, useFormalName) {
+  if (useFormalName) {
+    return "Lord" + firstname + " " + surname;
+  } else {
+    return firstname + " " + surname;
+  }
+}
+
+// But what if the person is a woman? Describe how you would fix the getFullname so it also works for women?
+
+// //  function getFullname(firstname, surname, gender, useFormalName) {
+// if(useFormalName){
+//   if(gender='female'){
+//     return "Lady" + firstname + " " + surname;}
+//   else{ return "Lord" + firstname + " " + surname;}
+// } else {
+//   return firstname + " " + surname;}
+// }
+
+// Event application 
+function getEventWeekday(dayToEvent){
+  const days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+  const today = new Date()
+  const todayIndex = today.getDay()
```

Comment:
This can be written a bit shorter considering that you don't use `today` anywhere else:
```suggestion
  const todayIndex = new Date().getDay();
```